src/target/source/codegen_cuda.cc

```diff
@@ -581,7 +581,10 @@ void CodeGenCUDA::VisitStmt_(const AllocateNode* op) {
   int32_t constant_size = op->constant_allocation_size();
   ICHECK_GT(constant_size, 0) << "Can only handle constant size stack allocation for now";
   const VarNode* buffer = op->buffer_var.as<VarNode>();
-  std::string scope = alloc_storage_scope_.at(buffer);
+  auto it = alloc_storage_scope_.find(buffer);
+  ICHECK(it != alloc_storage_scope_.end())
```

Comment:
I think we could be a little more specific here. We need an `AttrStmt` with a key of `storage_scope` right? So maybe say `"Buffer " << op->buffer_var << " is missing an AttrStmt with a \"storage_scope\" key"`.